suclass/cms/templates/body-post.html

```diff
@@ -1 +1,12 @@
-<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=8ca24832-80a4-4e66-ba71-c44c36c96b85"></script>
+<script>
+	$(document).ready(
+		setTimeout(
+			function()
+			{
+				var s = document.createElement("script");
+				s.id = "ze-snippet"
+				s.src = "https://static.zdassets.com/ekr/snippet.js?key=8ca24832-80a4-4e66-ba71-c44c36c96b85";
+				$("head").append(s)
```

Comment:
I'm not sure how much it matters, but since we're already adding this script to the BODY, let's continue doing so here, instead of adding it to HEAD.